src/app_engine/apprtc.py

```diff
@@ -560,16 +563,24 @@ def get(self, room_id):
         self.write_response('full_template.html')
         return
     # Parse out room parameters from request.
-    params = get_room_parameters(self.request, room_id, None, None)
+    loadingParams = get_room_parameters(self.request, room_id, None, None)
+    def maybe_include_loopback():
+      if loadingParams['isLoopback'] is True:
+        print 'here2'
+        return '<script src="/js/loopback.js"></script>'
```

Comment:
This is ugly. Have the template do this if condition:

{% if include_loopback_js }
 <script src="/...."></script>
{% endif }